q2_types/profile_hmms/_type.py

```diff
@@ -0,0 +1,136 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from qiime2.plugin import SemanticType
+from q2_types.plugin_setup import plugin
+from q2_types.profile_hmms._format import (
+    PressedProfileHmmsDirectoryFmt,
+    DnaSingleProfileHmmDirectoryFmt,
+    DnaMultipleProfileHmmDirectoryFmt,
+    RnaSingleProfileHmmDirectoryFmt,
+    RnaMultipleProfileHmmDirectoryFmt,
+    ProteinSingleProfileHmmDirectoryFmt,
+    ProteinMultipleProfileHmmDirectoryFmt
+)
+
+
+ProfileHMM = SemanticType('ProfileHMM', field_names='type')
```

Comment:
Please change this name (and all the corresponding classes, etc.) to `HMMProfile` so that it reads more naturally.

Reply:
Hey @misialq :). The correct name is "profile hidden Markov model", not  "hidden Markov model profile"